projects/ion/src/lib/indicator/indicator.component.html

```diff
@@ -0,0 +1,54 @@
+<div data-testid="ion-indicator" class="container">
+  <header class="title">
+    <h1 data-testid="ion-indicator-title">{{ title }}</h1>
+    <ion-icon
+      data-testid="ion-indicator-tooltip"
+      *ngIf="tooltipText"
+      type="exclamation-rounded"
+      size="16"
+      ionTooltip
+      ionTooltipPosition="bottomLeft"
+      ionTooltipColorScheme="dark"
+      ionTooltipTrigger="hover"
+      [ionTooltipTitle]="tooltipText"
+      [ionTooltipArrowPointAtCenter]="true"
+    >
+    </ion-icon>
+  </header>
+  <div class="content">
+    <h4 data-testid="ion-indicator-value">{{ value }}</h4>
+    <p data-testid="ion-indicator-second-value">{{ secondValue }}</p>
+  </div>
+  <footer data-testid="ion-indicator-footer" *ngIf="buttonConfig">
+    <ion-button
+      data-testid="ion-indicator-button-emitter"
+      *ngIf="buttonConfig.type === 'emitter'"
+      type="ghost"
+      size="sm"
+      rightSideIcon="true"
+      [label]="buttonConfig.label"
+      [iconType]="buttonConfig.icon ? buttonConfig.icon : 'right2'"
+      (ionOnClick)="emitButtonClick()"
+    ></ion-button>
+    <ion-button
+      data-testid="ion-indicator-button-redirect"
+      *ngIf="buttonConfig.type == 'redirect'"
+      type="ghost"
+      size="sm"
+      rightSideIcon="true"
+      [label]="buttonConfig.label"
+      [iconType]="buttonConfig.icon ? buttonConfig.icon : 'right2'"
+      (ionOnClick)="redirectTo()"
+    ></ion-button>
+    <ion-button
+      data-testid="ion-indicator-button-modal"
+      *ngIf="buttonConfig.type == 'modal'"
+      type="ghost"
+      size="sm"
+      rightSideIcon="true"
+      [label]="buttonConfig.label"
+      [iconType]="buttonConfig.icon ? buttonConfig.icon : 'right2'"
+      (ionOnClick)="openModal()"
+    ></ion-button>
```

Comment:
This could be just one button and you could check button type before taking an action inside your component. Something like this:
```suggestion
    <ion-button
      [attr.data-testid]="'ion-indicator-button-' + buttonConfig.type"
      type="ghost"
      size="sm"
      rightSideIcon="true"
      [label]="buttonConfig.label"
      [iconType]="buttonConfig.icon ? buttonConfig.icon : 'right2'"
      (ionOnClick)="action()"
    ></ion-button>
```
Then action would be a function that checks buttonType and call the respective action from that type of button.